src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -101,6 +101,18 @@ public partial struct ObjectEnumerator : System.Collections.Generic.IEnumerable<
             System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         }
     }
+    public readonly partial struct JsonEncodedText : System.IEquatable<System.Text.Json.JsonEncodedText>
```

Comment:
Bike-shedding: Why JsonEncodedText over JsonEncodedString? The JSON spec talks about strings, not text http://json.org/

Reply:
We discussed the names in the api-review and agreed on this over the alternatives (which included JsonEncodedString). I don't recall any strong reasoning one way or another, other than avoiding overloading "string" (i.e. `System.String`).